filters/auth/auth.go

```diff
@@ -31,3 +60,21 @@ func (authFilter AuthenticationFilter) basicAuth(req *web.Request, handler web.H
 
 	return handler(req)
 }
+
+// Filter which authenticates requests coming from Service Manger CLI using OAuth
```

Comment:
Do not mention SM CLI in the doc here. It might be a user request from a POSTman